actiontext/lib/action_text/fixture_set.rb

```diff
@@ -1,7 +1,57 @@
 # frozen_string_literal: true
 
 module ActionText
+  # \Fixtures are a way of organizing data that you want to test against; in
```

Comment:
Is this intentional or a typo?

```suggestion
  # Fixtures are a way of organizing data that you want to test against; in
```

Reply:
It's lifted from the ActiveRecord::FixtureSet class doc:

https://github.com/rails/rails/blob/76c834bcb67d947e22a126071d12d98cda511faf/activerecord/lib/active_record/fixtures.rb#L18

I'd appreciate any and all rdoc syntax guidance!